src/ios/BLECentralPlugin.m

```diff
@@ -509,7 +508,7 @@ - (CBPeripheral*)findPeripheralByUUID:(NSString*)uuid {
 
     for (CBPeripheral *p in peripherals) {
 
-        NSString* other = CFBridgingRelease(CFUUIDCreateString(nil, p.UUID));
+        NSString* other = CFBridgingRelease((__bridge CFTypeRef __nullable)(p.identifier.UUIDString));
```

Comment:
I had to change this line to:

```
NSString* other = p.identifier.UUIDString;
```

Otherwise things were crashing on connect, Using XCode 7 beta and iOS 8.3 on iPhone 6